packages/wasm/crate/src/tx.rs

```diff
@@ -341,6 +342,13 @@ pub async fn transaction_info_inner(
                 address_views.insert(address.encode_to_vec(), address_view);
                 asset_ids.insert(note.asset_id());
             }
+            ActionView::ActionDutchAuctionSchedule(ActionDutchAuctionScheduleView {
+                action,
+                ..
+            }) => {
+                asset_ids.insert(action.description.output_id);
+                asset_ids.insert(action.description.input.asset_id);
+            }
```

Comment:
This is a part of a match statement inside `transaction_info_inner`, in which we iterate over each action in the transaction, adding relevant pieces of information to the transaction perspective that will be needed to generate the transaction view. In this case, we need to add the asset IDs used for the auction to `asset_ids`, so that later on in this function, we can fetch those assets' metadata.